dashboard/src/App.css

```diff
@@ -0,0 +1,85 @@
+/* Basic container styles */
```

Comment:
What do you think of using scss so we can have variables, mixins and nested rules ? We are using it on cloud already